docs/README.md

```diff
@@ -1,91 +1,192 @@
-| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
-
-Prop types are: 
-
-- `string` 
-- `enum` 
-- `number` 
-- `boolean` 
-- `object` 
-- `array` 
-
-When documenting a prop whose type is `object` or `array` another prop table will be needed. You can create it following the example below:
-
-- `propName` object:
-
-| Prop name    | Type            | Description    | Default value                                                                                                                               |
-| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
-| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
-
-
-Remember to also use this Configuration section to  **showcase any necessary disclaimer** related to the app and its blocks, such as the different behavior it may display during its configuration. 
-
-## Modus Operandi *(not mandatory)*
-
-There are scenarios in which an app can behave differently in a store, according to how it was added to the catalog, for example. It's crucial to go through these **behavioral changes** in this section, allowing users to fully understand the **practical application** of the app in their store.
-
-If you feel compelled to give further details about the app, such as it's **relationship with the VTEX admin**, don't hesitate to use this section. 
+The Product Specifications app provides blocks to render product specification data.
+
+![Product Specifications Example](https://user-images.githubusercontent.com/284515/90801619-859c6580-e2ec-11ea-9a7d-6e80d8f3d315.png)
+
+## Configuration
+
+### Step 1 - Adding the Product Specifications app to your theme's dependencies
+
+In your theme's `manifest.json`, add the Product Specification app as a dependency:
+
+```json
+"dependencies": {
+  "vtex.product-specification": "1.x"
+}
+```
+
+Now, you can use all the blocks exported by the `product-specifications` app. Check out the full list below:
+
+| Block name                    | Description                                                                                                     |
+| ----------------------------- | --------------------------------------------------------------------------------------------------------------- |
+| `product-specification-group` | Renders its children for every product specification group.                                                     |
+| `product-specification`       | Renders its children for every product specification. It should be placed bellow `product-specification-group`. |
+| `product-specification-value` | Renders its children for every product value. It should be placed bellow `product-specification`.               |
+| `product-specification-text`  | Renders the data of a specification group, specification, and specification value.                              |
```

Comment:
I didn't understand the blocks' description (therefore I was not able to give more details on step 2). 

In my opinion, the current table is too focused on explaining the relationship between blocks. I mean, it is important to do that, but I think the focus here should be to explain the block functionality so users can properly build the template in the next step.

Reply:
@guerreirobeatriz 

A Product Specification it's structured like this:

- Product
  - List of Specifications Groups ¹
    - Group
      - Group Name ²
      - List of Specifications ¹
        - Specification
          - Specification Name ²
          - List of Specification Values ¹
            - Specification Value ²

This app iterate through each list (¹) to render it's values (²) with the block `product-specification-text`